packages/transport/src/create.ts

```diff
@@ -142,14 +119,17 @@ export const createChannelTransport = ({
           else throw Error('Unknown transport item');
         } catch (error) {
           const err = ConnectError.from(error);
-          // throwing from a listener is pointless. store the first error, log
+          // some fundamental disaster has occurred, and now the transport is
+          // broken forever. we have no specific request to reject, but throwing
+          // from a listener is a dead end. so we store it to throw at every
+          // subsequent interaction.
           listenerError ??= err;
-          console.error('Error in transport listener', err);
+          console.warn('Transport failed', err);
```

Comment:
If it's a fundamental disaster kind of condition, think console.error is probably appropriate